task/upload.go

```diff
@@ -117,24 +163,28 @@ func getFileUrl(os *api.ObjectStore, cfg ImportTaskConfig, params api.UploadTask
 		u.Path = path.Join(u.Path, params.UploadedObjectKey)
 		return u.String(), nil
 	}
-	if params.URL == "" {
+	return getFileUrl(cfg, params.URL)
+}
+
+func getFileUrl(cfg ImportTaskConfig, URL string) (string, error) {
```

Comment:
**nit:** Local variables are generally lower case, so I think this would feel more natural.
```suggestion
func getFileUrl(cfg ImportTaskConfig, url string) (string, error) {
```

Reply:
Updated